man/ippeveprinter.1

```diff
@@ -137,6 +140,9 @@ printer command ("\-c ippeveps").
 Specifies the maximum IPP version to report.
 2.0 is the default.
 .TP 5
+\fB-a \fIfilename.conf\fR
+Load printer attributes from conf file acquired by parameter "--ippserver" of command "ipptool". See "man ipptool" for more info.
```

Comment:
Try:
This needs to be properly annotated for the cross-reference links to work. Also, the man page guidelines say to put one sentence per line. How about:

```
Loads printer attributes from "filename.conf".
Attribute files can be created using the
.BR ipptool (1)
command with the
.I \-\-ippserver
option.
```

Reply:
Applied.